agent/bench.py

```diff
@@ -751,9 +755,18 @@ def _start(self):
 
     @job("Force Update Bench Limits")
     def force_update_limits(self, memory_high, memory_max, memory_swap, vcpu):
-        self._stop()
-        self._update_runtime_limits(memory_high, memory_max, memory_swap, vcpu)
-        self._start()
+        def _update_limits(should_start: bool):
+            self._stop()
+            self._update_runtime_limits(memory_high, memory_max, memory_swap, vcpu)
+            if should_start:
+                self._start()
```

Comment:
should probably add  this to start queue

a case: suppose this bench was shut down due to this update limit and at that same time a request came to start another bench, and that was started because this bench left enough room in the server for that to run